app/services/support_interface/candidate_autofill_usage_export.rb

```diff
@@ -0,0 +1,157 @@
+module SupportInterface
+  class CandidateAutofillUsageExport
+    def data_for_export
+      degree_grade_output +
+        degree_institution_output +
+        degree_subject_output +
+        degree_type_output +
+        other_grade_output +
+        other_type_output
+    end
+
+  private
+
+    def other_type_output
```

Comment:
I think it would read better if these methods followed the same order that they are called in above. Or reverse order even, but something systematic.

Reply:
👍